mpl_interactions/widgets.py

```diff
@@ -1,15 +1,10 @@
+"""Custom matplotlib widgets made for use in this library."""
+
 from matplotlib.cbook import CallbackRegistry
 from matplotlib.widgets import AxesWidget
 from matplotlib import cbook, ticker
 import numpy as np
 
-__all__ = [
-    "scatter_selector",
-    "scatter_selector_index",
-    "scatter_selector_value",
-    "RangeSlider",
-]
-
```

Comment:
any reason to remove this?

Reply:
`__all__` determines which functions are rendered in the API by sphinx-apidoc (and I guess autosummary too). If it's removed, all public members are included. The drawback: `SliderBase` is now imported as well if you use `from mpl_interactions.widgets import *`. That `*` syntax should be avoided though.

Why did I remove it, and include `SliderBase`? Sphinx-apidoc generates links to the base classes as well:
![image](https://user-images.githubusercontent.com/29308176/126540791-9aeeb626-4fee-4f48-94c2-26c424e0dba2.png)

If those base classes are not available in the API, Sphinx will just raise some warnings. But I think those warnings should be avoided (hence 0efc4e0 and aafbcb0), so I included the `SliderBase` class. As an alternative, we can list it under [`nitpicky_ignore`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-nitpick_ignore) and put back the `__all__`.